### ISSUE TITLE
DAC 21 - Medium Priority WCAG Level AA - Colour Contrast [Non-Text] 08

### ISSUE THREAD

**Issue by JadeneAd** [ONSvisual/income-scrolly#21]
<img width="353" alt="DAC_Colour_Contrast_Non_Text_08" src="https://user-images.githubusercontent.com/75850106/155561923-bc5487a8-d784-4a7e-959d-84daf8e5363d.png">

### Issue
A black line is used to outline the area of a place, however; the lines’ contrast against adjacent colours may not be obvious for low vision users to see. The black line against adjacent colours is less than the 3:1 minimum.

Foreground: #000000 
Background: #053061
The contrast ratio is: 1.6:1 
1.4.11 Non-text Contrast (AA)
Fail for UI components and graphical objects

<img width="602" alt="DAC 21 Current code ref" src="https://user-images.githubusercontent.com/75850106/155562071-56edeacc-80e0-4da9-9faf-a8c8adad1c14.png">

### Suggested Solution

Ensure that the black line has a contrast ratio of 3:1 or greater.
